tests/queries/0_stateless/02812_from_to_utc_timestamp.sh

```diff
@@ -18,15 +18,15 @@ ${CLICKHOUSE_CLIENT} -q "INSERT INTO test_tbl values(2, '2023-03-16 11:22:33', '
 ${CLICKHOUSE_CLIENT} -q "INSERT INTO test_tbl values(3, '2023-03-16 11:22:33', '2023-03-16 11:22:33.123456')"
 $CLICKHOUSE_CLIENT -q "select x, to_utc_timestamp(toDateTime('2023-03-16 11:22:33'), 'Etc/GMT+1'), from_utc_timestamp(toDateTime64('2023-03-16 11:22:33', 3), 'Etc/GMT+1'), to_utc_timestamp(y, 'Asia/Shanghai'), from_utc_timestamp(z, 'Asia/Shanghai') from test_tbl order by x"
 # timestamp convert between DST timezone and UTC
-$CLICKHOUSE_CLIENT "${client_opts[@]}" -q "select to_utc_timestamp(toDateTime('2024-02-24 11:22:33'), 'Europe/Madrid'), from_utc_timestamp(toDateTime('2024-02-24 11:22:33'), 'Europe/Madrid')"
-$CLICKHOUSE_CLIENT "${client_opts[@]}" -q "select to_utc_timestamp(toDateTime('2024-10-24 11:22:33'), 'Europe/Madrid'), from_utc_timestamp(toDateTime('2024-10-24 11:22:33'), 'Europe/Madrid')"
-$CLICKHOUSE_CLIENT "${client_opts[@]}" -q "select to_utc_timestamp(toDateTime('2024-10-24 11:22:33'), 'EST'), from_utc_timestamp(toDateTime('2024-10-24 11:22:33'), 'EST')"
+$CLICKHOUSE_CLIENT -q "select to_utc_timestamp(toDateTime('2024-02-24 11:22:33'), 'Europe/Madrid'), from_utc_timestamp(toDateTime('2024-02-24 11:22:33'), 'Europe/Madrid') ${timezone_setting[@]}"
```

Comment:
I think randomizing `session_timezone` makes sense, however you are not passing any timezone argument to `toDateTime`. I think the proper way should be:
```suggestion
$CLICKHOUSE_CLIENT -q "select to_utc_timestamp(toDateTime('2024-02-24 11:22:33', 'Europe/Madrid'), 'Europe/Madrid'), from_utc_timestamp(toDateTime('2024-02-24 11:22:33', 'Europe/Madrid'), 'Europe/Madrid')"
```

What do you think?

Reply:
We cannot do this:
`Code: 43. DB::Exception: Received from localhost:9000. DB::Exception: Function toUTCTimestamp's 1st argument should not have explicit time zone.: In scope SELECT 'leap year:', to_utc_timestamp(toDateTime('2024-02-28 23:22:33', 'Europe/Madrid'), 'EST'), from_utc_timestamp(toDateTime('2024-03-01 00:22:33', 'Europe/Madrid'), 'EST'). (ILLEGAL_TYPE_OF_ARGUMENT)`